hail/python/hail/setup.py

```diff
@@ -2,8 +2,44 @@
 
 from setuptools import setup, find_packages
 
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+# https://stackoverflow.com/questions/45150304/how-to-force-a-python-wheel-to-be-platform-specific-when-building-it
+try:
+    from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+    class bdist_wheel(_bdist_wheel):
+        def finalize_options(self):
+            _bdist_wheel.finalize_options(self)
+            self.root_is_pure = False
+except ImportError:
+    bdist_wheel = None
+
 setup(
     name="hail",
-    version="0.2",
-    packages=find_packages()
+    version="0.2.10",
+    author="Hail Team",
+    author_email="hail-team@broadinstitute.org",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://hail.is",
+    packages=find_packages(),
+    package_data={'': ['hail-all-spark.jar']},
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+    ],
+    install_requires=[
+        'numpy<2',
```

Comment:
We can set lower bounds too! I just want to avoid equality because, if people correctly use semantic versioning, we should be able to benefit from backwards compatible improvements to the library.

Reply:
I added pandas>0.22